sdk/dotnet/AppEngine/V1/App.cs

```diff
@@ -75,6 +75,12 @@ public partial class App : Pulumi.CustomResource
         [Output("iap")]
         public Output<Outputs.IdentityAwareProxyResponse> Iap { get; private set; } = null!;
 
+        /// <summary>
+        /// Identifier of the Application resource. This identifier is equivalent to the project ID of the Google Cloud Platform project where you want to deploy your application. Example: myapp.
+        /// </summary>
+        [Output("id")]
+        public Output<string> Id { get; private set; } = null!;
```

Comment:
You can see the warning from the C# compiler about `Id` overriding the base property incorrectly.